app/scripts/lib/setupSentry.js

```diff
@@ -140,23 +141,7 @@ export default function setupSentry({ release, getState }) {
     ],
     release,
     beforeSend: (report) => rewriteReport(report, getState),
-    beforeBreadcrumb(breadcrumb) {
```

Comment:
Are there any desktop specific reasons that this `beforeBreadcrumb` method was changed?

Reply:
This is exported just to avoid duplicating the same code in the app where we also need to setup Sentry.

https://github.com/MetaMask/metamask-desktop/blob/main/packages/app/src/app/globals.ts#L130